src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix an issue where the `match-results-forward` command generates Sarif with duplicated rules if there are results with sub `ruleId`. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486)
```

Comment:
>Fix an issue where the `match-results-forward` command generates Sarif with duplicated rules if there are results with sub `ruleId [](http://example.com/codeflow?start=10&length=130)

Please change this to: 

Remove duplicated rule definitions when executing `match-results-forward` commands for results with sub-rule ids.

Reply:
updated